style.css

```
@@ -20,10 +29,49 @@ nav { line-height: normal; }
 
 :root {
     --header-height: 3em;
+    /* custom */
     --header-bg: #f8f8f8;
+    --body-bg: white;
+    --code-bg: #f3f3f3;
+    --text-color: black;
+    --link-color: hsl(210, 100%, 30%);
+
+    --implicit-arg-text-color: var(--text-color);
+    
+    --def-color: #92dce5;
+    --def-color-hsl-angle: 187;
+    --theorem-color: #8fe388;
+    --theorem-color-hsl-angle: 115;
+    --axiom-and-constant-color: #f44708;
+    --axiom-and-constant-color-hsl-angle: 16;
+    --structure-and-inductive-color: #f0a202;
+    --structure-and-inductive-color-hsl-angle: 40;
+    --starting-percentage: 100;
+    
     --fragment-offset: calc(var(--header-height) + 1em);
     --content-width: 55vw;
 }
+
+[data-theme="dark"] {
```

```suggestion
[data-theme="dark"] {
    color-scheme: dark;

```
which changes the scrollbars to dark too